src/build_function.jl

```diff
@@ -406,7 +408,7 @@ function numbered_expr(O::Symbolic,varnumbercache,args...;varordering = args[1],
 end
 
 function numbered_expr(de::Equation,varnumbercache,args...;varordering = args[1],
-                       lhsname=gensym("du"),rhsnames=[gensym("MTK") for i in 1:length(args)],offset=0)
+                       lhsname=:du,rhsnames=[Symbol("MTK$i") for i in 1:length(args)],offset=0)
```

Comment:
These are only used in C and MATLAB outputs iirc

Reply:
Might as well make the codegen stable there too.

Reply:
And in that case, we really don't need to gensym since it's not going to take globals from Julia 😅